src/js/themes/microvideo.js

```diff
@@ -102,7 +102,7 @@ template.innerHTML = /*html*/`
     align-items: start;
   }
 
-  :host([control-bar-place$="end"]) media-control-bar {
+  :host([controlbarplace$="end"]) media-control-bar {
```

Comment:
NOTE: Should discuss - normalize so official attrs for official themes are also lowercase/"smushedcase" (See below for at least one particular friction point/tradeoff).